### ISSUE TITLE
Course Review

### ISSUE THREAD

**Issue by HuwWDay** [Bristol-Training/intro_to_AI#7]
See the prerequisites of the course [here](https://github.com/Bristol-Training/intro_to_AI/blob/main/0-Introduction_LOs.ipynb)

This is intended to be delivered as we would other Python training sessions, ideally in a 3 hour slot (but I suspect this course might be a bit too long!) 

I'd really appreciate if you could have a read through the course notes and give any comments/feedback (you could just reply in this issue) on how things have been explained. If you have any suggested changes could you please make them in a new branch. See below a table of contents and links, please note the "part 2" bits are still under development:

- [ ] [Intro](https://github.com/Bristol-Training/intro_to_AI/blob/main/0-Introduction_LOs.ipynb)

Theory Part 1:
- [ ] [The Perceptron](https://github.com/Bristol-Training/intro_to_AI/blob/main/TheoryPart1/TheoryPart1_BoyiFINAL.ipynb)
- [ ] [Multiple Layers and Activation Functions](https://github.com/Bristol-Training/intro_to_AI/blob/main/TheoryPart1/2-HiddenLayers_BoyiFINAL.ipynb)
- [ ] [Evaluating Loss](https://github.com/Bristol-Training/intro_to_AI/blob/main/TheoryPart1/3-EvaluatingOurMLP.ipynb)

Code Part 1:
- [ ] [Pytorch and Tensors](https://github.com/Bristol-Training/intro_to_AI/blob/main/CodePart1/1-Pytorch_and_Tensors_zhengzhe.ipynb)
- [ ] [Classes and Perceptron](https://github.com/Bristol-Training/intro_to_AI/blob/main/CodePart1/2-Classes_and_Perceptron_zhengzhe.ipynb)
- [ ] [Data Pipeline](https://github.com/Bristol-Training/intro_to_AI/blob/main/CodePart1/3-DataPipeline_zhengzhe.ipynb)
- [ ] [Multi-layer Perceptron Implementation](https://github.com/Bristol-Training/intro_to_AI/blob/main/CodePart1/4-Multi_Layer_Perceptron_zhengzhe.ipynb)

Theory Part 2:

- [ ] [Gradient Descent](https://github.com/Bristol-Training/intro_to_AI/blob/main/TheoryPart2/1-Gradient%20Descent.ipynb)
- [ ] [Back propogation](https://github.com/Bristol-Training/intro_to_AI/blob/main/TheoryPart2/2-Back%20propagation.ipynb)

Code Part 2:

- [ ] [Training an MLP](https://github.com/Bristol-Training/intro_to_AI/blob/main/CodePart2/5-Training-Pipeline.ipynb)
- [ ] [Interpreting the learning rates](https://github.com/Bristol-Training/intro_to_AI/blob/main/CodePart2/6-learning-rate-interpret.ipynb)
- [ ] [Local vs HPC Processing](https://github.com/Bristol-Training/intro_to_AI/blob/main/CodePart2/7-brief-HPC.ipynb)

**Comment by jatonline** [comment]
@HuwWDay I updated the link to '[Multiple Layers and Activation Functions](https://github.com/Bristol-Training/intro_to_AI/blob/main/TheoryPart1/2-HiddenLayers_BoyiFINAL.ipynb)', as it went to the '[The Perceptron](https://github.com/Bristol-Training/intro_to_AI/blob/main/TheoryPart1/TheoryPart1_BoyiFINAL.ipynb)' instead. Is this the right link now?

**Comment by jatonline** [comment]
Also updated [Data Pipeline](https://github.com/Bristol-Training/intro_to_AI/blob/main/CodePart1/3-DataPipeline_zhengzhe.ipynb) and [Back propogation](https://github.com/Bristol-Training/intro_to_AI/blob/main/TheoryPart2/2-Back%20propagation.ipynb), whilst I'm here :)

**Comment by HuwWDay** [comment]
[heart]         Huw Day reacted to your message:
________________________________
